pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -281,7 +281,7 @@ func (m *kubeGenericRuntimeManager) startContainer(podSandboxID string, podSandb
 		if handlerErr != nil {
 			klog.ErrorS(handlerErr, "Failed to execute PostStartHook", "pod", klog.KObj(pod),
 				"podUID", pod.UID, "containerName", container.Name, "containerID", kubeContainerID.String())
-			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, msg)
+			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, "PostStartHook failed")
```

Comment:
```suggestion
			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, fmt.Sprintf("PostStartHook failed: %s", msg))
```
I think it is best to attach the `msg`

https://github.com/kubernetes/kubernetes/blob/e1acbbd8fde3505423df6c336a4d852bfc50e49d/pkg/kubelet/lifecycle/handlers.go#L59-L83

Reply:
@wzshiming the maintainers specifically requested that the response body of the lifecycle handler be removed from the events: https://github.com/kubernetes/kubernetes/pull/86139#discussion_r623504269

Reply:
Thanks, got it.